packages/oauth/src/providers/apple.ts

```diff
@@ -137,8 +144,11 @@ const getAppleUser = (idToken: string): AppleUser => {
 	const jwtPayload = decodeIdToken<AppleUser>(idToken);
 	return {
 		email: jwtPayload.email,
-		email_verified: jwtPayload.email_verified,
```

Comment:
`emaiLverified` is included in the ID token

https://developer.apple.com/documentation/sign_in_with_apple/sign_in_with_apple_rest_api/authenticating_users_with_sign_in_with_apple#3383773